ruby-challenges.rb

```diff
@@ -10,23 +10,76 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def array_of_words (array, letter)
+    returns_array = []
+    array.each do |string|
+    if string.include(letter)
+    end
+    
+end
+
```

Comment:
You are missing an `end` here. You are also missing an output for your conditional statement.